### DISCUSSION TITLE
添加子模块自动部署后网页一片空白

### DISCUSSION THREAD

**Discussion by AuntYang** [xaoxuu/hexo-theme-stellar#282] [Q&A]
我按照官方wiki的教程，将主题添加为子模块
![image](https://user-images.githubusercontent.com/49216233/229332432-0b1a8313-e75f-476e-b556-bc3c2631ed89.png)
然后GitHub action自动部署，提示没有layout
![image](https://user-images.githubusercontent.com/49216233/229332502-d3d4651c-0935-4c04-b206-be9c2a6bcdfc.png)
生成的静态文件
![image](https://user-images.githubusercontent.com/49216233/229332518-7600d6f5-951e-42ca-b86f-97ce6a6b5541.png)
index空白
![image](https://user-images.githubusercontent.com/49216233/229332565-d0a5650e-1894-4465-ae45-ce2b8117d507.png)
已经阅读完所有issues，但没能找到类似问题，请问我该从何排查问题？

**Comment by xaoxuu** [comment]
这种问题一般是主题资源没有拉到
